homework-submission/react-hyf/react-hyf-server/server.js

```diff
@@ -0,0 +1,14 @@
+const express = require("express");
+const app = express();
+
+app.use(express.static("/home/exilon/HYF/react/react-hyf/react-hyf-client/public"));
```

Comment:
This line will only work on your machine, I device doesn't have the username of `exilon`.

And for the sendFile too! you are specifying the current directory as a root, and add to it `/react-hyf-client/build/index.html` which could be for you computer as:
`/home/exilon/HYF/react/react-hyf/react-hyf-server/react-hyf-client/build/index.html`

Have you tried them?